src/solvers/flattening/boolbv_div.cpp

```diff
@@ -10,56 +10,58 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include <util/std_types.h>
 
-bvt boolbvt::convert_div(const div_exprt &expr)
+bvt boolbvt::convert_div(const div_exprt &div_expr)
 {
-  if(expr.type().id()!=ID_unsignedbv &&
-     expr.type().id()!=ID_signedbv &&
-     expr.type().id()!=ID_fixedbv)
-    return conversion_failed(expr);
+  if(
+    div_expr.type().id() != ID_unsignedbv &&
+    div_expr.type().id() != ID_signedbv && div_expr.type().id() != ID_fixedbv)
+    return conversion_failed(div_expr);
 
-  std::size_t width=boolbv_width(expr.type());
+  std::size_t bv_width = boolbv_width(div_expr.type());
 
-  if(width==0)
-    return conversion_failed(expr);
+  if(bv_width == 0)
+    return conversion_failed(div_expr);
 
-  if(expr.op0().type().id()!=expr.type().id() ||
-     expr.op1().type().id()!=expr.type().id())
-    return conversion_failed(expr);
+  if(
+    div_expr.dividend().type().id() != div_expr.type().id() ||
+    div_expr.divisor().type().id() != div_expr.type().id())
+    return conversion_failed(div_expr);
 
-  bvt op0=convert_bv(expr.op0());
-  bvt op1=convert_bv(expr.op1());
+  bvt dividend = convert_bv(div_expr.dividend());
+  bvt divisor = convert_bv(div_expr.divisor());
 
```

Comment:
These two could likely be const.

Reply:
Afraid not: later in the function we add fraction and least-significant bits to those, respectively.